magento/inventory
# Out of stock label is not working correctly for grouped type product.

### Description (*)


### Fixed Issues (if relevant)

1. none.

### Manual testing scenarios (*)

1. Create grouped product wit two options on custom stock.
2. Navigate to storefront and buy all qty of one of the grouped product options.
3. Place order and clean cache.
4. Verify sold out option is not visible on grouped PDP.

### Questions or comments


### Contribution checklist (*)
